### PR TITLE
removed push

### PR DESCRIPTION
**Pull request by aagavin**
Removed push notification code

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -75,13 +71,6 @@ export class MyApp {
     ];
 
     const authObserver = af.auth.subscribe(user => {
```
Comment (LeontiBrechko) [inline src/app/app.component.ts]: Do we still need this subscription?

Comment (aagavin) [reply src/app/app.component.ts]: Yea I don't think it does anything

Comment (aagavin) [reply src/app/app.component.ts]: will remove

**Review by MavOmidK — APPROVED** [review]
checked it out works gud

**Pull request merged**